### DISCUSSION TITLE
驗證碼錯誤手動解決辦法

### DISCUSSION THREAD

**Discussion by MagicalLeo** [Mufanc/iSmartAuto#51] [General]
在.\iSmartAuto\automaton\spider\spider.py文件底下找到這行
result = await self.get('https://sso.ismartlearning.cn/captcha.html')
在這行後面加入下面這幾行之後便可以手動輸入驗證碼

            result = result.content
            bytes_stream = BytesIO(result)
            roiimg = Image.open(bytes_stream)
            roiimg.show()  #展示图片
            print(type(result))
            input(code)

前面記得import這幾條:
from io import BytesIO
from PIL import Image
如果顯示no module那就在cmd裡
pip install pillow

**Comment by MagicalLeo** [comment]
加了之後還是顯示驗證碼錯誤
我懷疑是
result = await self.get('https://sso.ismartlearning.cn/captcha.html')
這句取得的驗證碼圖片不是同一張驗證碼圖片，希望能有人解答

**Comment by Superaceno1** [comment]
手动取得验证码后在哪里输入验证码呢

**Reply by MagicalLeo** [reply]
直接打在terminal終端那裡就好了吧~

**Reply by MagicalLeo** [reply]
就是你運行程式的那個視窗那裡輸入就好了，但我自己用了還是有問題，好像是他這個程式取得的驗證碼不是你登入時需要的驗證碼
